client/my-sites/themes/single-site.jsx

```diff
@@ -184,7 +184,7 @@ export default connect(
 		{
 			queryParams: ThemesListSelectors.getQueryParams( state ),
 			themesList: ThemesListSelectors.getThemesList( state ),
-			selectedSite: getSelectedSite( state )
+			selectedSite: getSelectedSite( state ) || false,
```

Comment:
> question: why can't we rely on what the state gives back?

Because the [selector returns null](https://github.com/Automattic/wp-calypso/blob/9a5a3d7b79593839e962802b72700a36bdecf7a1/client/state/ui/selectors.js#L9) if the site data is not loaded, whereas the showcase code was written when `sites.getSelectedSite()` returned `false`.